src/backend/utils/sanitize-html.js

```diff
@@ -7,6 +7,36 @@ module.exports = function(dirty) {
   return sanitizeHtml(dirty, {
     // Add <img> to the list of allowed tags, see:
     // https://github.com/apostrophecms/sanitize-html#what-are-the-default-options
-    allowedTags: sanitizeHtml.defaults.allowedTags.concat(['img', 'figure']),
+    allowedTags: sanitizeHtml.defaults.allowedTags.concat([
```

Comment:
This won't remove `<iframe>`, only add the tags in your list:

```js
> ['iframe', 'a', 'img'].concat(['img', 'figure', 'p'])
[ 'iframe', 'a', 'img', 'img', 'figure', 'p' ]
```

You would have to remove items from the original array (i.e., sanitizeHtml.defaults.allowedTags).  However, another approach, since you're listing them all manually below, would be to just define your own list:

```js
allowedTags: [
  // your list of tags here...
], 
```

If you're going to do it this way, please consolidate the list you have below with what's in https://github.com/apostrophecms/sanitize-html#what-are-the-default-options, since you will miss some otherwise.